content/pages/meetup/2023/0920-nycu.rst

```diff
@@ -0,0 +1,100 @@
+===============================================================
+Meetup 2023 September 20th in NYCU (do-unknown-things training)
+===============================================================
+
+:date: 2023-08-31 13:00
+:url: meetup/2023/0920-nycu
+:save_as: meetup/2023/0920-nycu.html
+
+
+Date
+-----
+
+* Date: September 20th, 2023
+* Time: 18:30 -- 21:30 (3 hours)
+
+|
+
+Agenda
+--------
+
+* 18:30-19:00 Check in and icebreaking
+* 19:00―19:10 Presentation: what are the things
+* 19:10―19:30 Presentation: how to get things done
+* 19:30―19:45 Discussion: things you have done
+* 19:45―20:00 Case study: sciwork meetup
+* 20:00―20:15 Discussion: analyze the things you want to do
+* 20:15―20:30 Hands-on: make the plan and review
+* 20:30-21:30 Meeting: prepare September scisprint/ Free discussion 
+
+|
+
+Subjects
+------------------
+
+Training: do unknown things
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+Speaker: 
+Yung-Yu - Engineer from Synopsis and enthusiastist of numerical code. He is also 
```

Comment:
@EN-Chou , you misspelled Synopsys.  Enthusiast was also misspelled.  Next time, please run spell checking more rigorously before you send it for review.

And I am not a founder of PyCon.  Please copy and edit what is in https://sciwork.dev/about#yung-yu-chen for the introduction.

Reply:
I apologize for making these mistakes and thanks for your reminder. I have copy the content https://sciwork.dev/about#yung-yu-chen and make a slight change in the opening of the introduction so as to align with other posts.

Reply:
Thanks.  It looks good.